pkg/cvo/status.go

```diff
@@ -158,7 +158,11 @@ func (optr *Operator) syncStatus(original, config *configv1.ClusterVersion, stat
 	// update the config with the latest available updates
 	if updated := optr.getAvailableUpdates().NeedsUpdate(config); updated != nil {
 		config = updated
-	} else if original == nil || original == config {
```

Comment:
Hrm, this isn't the same condition anymore.  If updated is not nil, original should not be deep copied no matter what.

Reply:
brought it back to previous state with https://github.com/openshift/cluster-version-operator/pull/243/commits/03e188b3d7fcf3e2e374687186d8c994685d605f